README.md

```diff
@@ -45,8 +45,8 @@ need to be manually installed. Example flow of installing Metal3 provider:
    level of the logging verbose with a positive integer number, ie. -v5.
 
    ```shell
-   clusterctl init --core cluster-api:v1.6.5 --bootstrap kubeadm:v1.6.5 \
-       --control-plane kubeadm:v1.6.5 -v5
+   clusterctl init --core cluster-api:v1.6.6 --bootstrap kubeadm:v1.6.6 \
+       --control-plane kubeadm:v1.6.6 -v5
```

Comment:
@adilGhaffarDev can you also update line 63 ? 

`clusterctl init --infrastructure metal3:v1.5.0`

Reply:
done